### PR TITLE
Fix issue 204

### PR DESCRIPTION
**Pull request by tuliom**

### CODE REVIEW

**Comment by tuliom** [comment]
AT 7.0 RHEL6 ppc64 85 failed due to disk space issues.
I retried that build as AT 7.0 RHEL6 ppc64 86 and it passed.

AT 6.0 RHEL6 ppc64 is being fixed in PR #222.
AT next SLES12 ppc64le and AT next Ubuntu16 ppc64le failed in the python testcases.

File: configs/11.0/packages/gcc/stage_optimized [hunk 1]
```diff
@@ -93,10 +93,15 @@ atcfg_make() {
...
+	# ssp_have_usable_vsnprintf=define is required to avoid configure
+	# scripts from running programs compiled with a -mcpu that is superior
+	# of the base value, which can cause a silent illegal instruction
+	# error, forcing the script to assume that vsnprintf is not usable.
 	PATH=${at_dest}/bin:${PATH} \
 	libgo_cv_lib_setcontext_clobbers_tls=no \
 		${SUB_MAKE} all \
-			    gcc_cv_have_tls=yes
+			    gcc_cv_have_tls=yes \
+			    ssp_have_usable_vsnprintf=define
```
Comment (gftg85) [inline configs/11.0/packages/gcc/stage_optimized]: It's fine to keep these variables here, but is there a reason to not keep them together with the other cached variable (libgo_cv_lib_setcontext_clobbers_tls)?

Comment (tuliom) [reply configs/11.0/packages/gcc/stage_optimized]: AFAIK, variables defined as env vars are not passed to subshells unless they're explicitly exported or if the underlying shell receives them.
I have already confirmed that ssp_have_usable_vsnprintf is affected by this.

**Comment by tuliom** [comment]
Merged.
This fixes issue #204 .

**Pull request closed without merging**